<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pokecraft</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../base/resources/resources.qrc">
    <normaloff>:/images/pokecraft.png</normaloff>:/images/pokecraft.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/images/interface/player/background.png);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/background.png);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>99</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_login_1">
          <item>
           <spacer name="horizontalSpacer_login_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QFrame" name="frame_login_2">
            <property name="minimumSize">
             <size>
              <width>777</width>
              <height>379</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>777</width>
              <height>379</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/images/savegame-select.png);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_login">
               <property name="minimumSize">
                <size>
                 <width>760</width>
                 <height>360</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>760</width>
                 <height>360</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/images/empty.png);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QScrollArea" name="scrollArea">
                  <property name="styleSheet">
                   <string notr="true">QScrollArea {
	background-image: url(:/images/light-white.png);
border: 1px solid #000;
}
QLabel { padding:2px; }</string>
                  </property>
                  <property name="widgetResizable">
                   <bool>true</bool>
                  </property>
                  <widget class="QWidget" name="scrollAreaWidgetContents">
                   <property name="geometry">
                    <rect>
                     <x>0</x>
                     <y>0</y>
                     <width>750</width>
                     <height>298</height>
                    </rect>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_5">
                    <item>
                     <widget class="QLabel" name="savegameEmpty">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#a0a0a0;&quot;&gt;Empty&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <spacer name="horizontalSpacer_11">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveGame_Play">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
     border-radius: 15px;
     border-color:#000;
     border: 1px outset #999;
     background-color: rgb(255, 255, 255);
     padding:1px 10px;
}
 QPushButton:pressed {
     background-color: rgb(240,240,240);
     border-style: inset;
 }</string>
                    </property>
                    <property name="text">
                     <string>Play selected game</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_12">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_login_4">
                  <item>
                   <spacer name="horizontalSpacer_login_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveGame_New">
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
     border-radius: 15px;
     border-color:#000;
     border: 1px outset #999;
     background-color: rgb(255, 255, 255);
     padding:1px 10px;
}
 QPushButton:pressed {
     background-color: rgb(240,240,240);
     border-style: inset;
 }</string>
                    </property>
                    <property name="text">
                     <string>New game</string>
                    </property>
                    <property name="flat">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveGame_Rename">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
     border-radius: 15px;
     border-color:#000;
     border: 1px outset #999;
     background-color: rgb(255, 255, 255);
     padding:1px 10px;
}
 QPushButton:pressed {
     background-color: rgb(240,240,240);
     border-style: inset;
 }</string>
                    </property>
                    <property name="text">
                     <string>Rename</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveGame_Copy">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
     border-radius: 15px;
     border-color:#000;
     border: 1px outset #999;
     background-color: rgb(255, 255, 255);
     padding:1px 10px;
}
 QPushButton:pressed {
     background-color: rgb(240,240,240);
     border-style: inset;
 }</string>
                    </property>
                    <property name="text">
                     <string>Copy</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="SaveGame_Delete">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
     border-radius: 15px;
     border-color:#000;
     border: 1px outset #999;
     background-color: rgb(255, 255, 255);
     padding:1px 10px;
}
 QPushButton:pressed {
     background-color: rgb(240,240,240);
     border-style: inset;
 }</string>
                    </property>
                    <property name="text">
                     <string>Delete</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_login_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_login_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>98</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../base/resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
